modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -204,14 +204,14 @@ definition {
 	test CanCreateEmailNotificationTemplate {
 		property portal.acceptance = "true";
 
-		task ("Given the user navigates to Notifications - Templates in the Control Panel") {
+		task ("Given the user navigates to Notifications Templates in the Control Panel") {
```

Comment:
This change shouldn't be part of this, it's a wording change so you should use a separate commit with the message `Wordsmith`